wildfirepy/net/util/__init__.py

```diff
@@ -1 +1 @@
-from wildfirepy.net.util.base import ModisHtmlParser, URLOpenerWithRedirect
+from .modis import *
```

Comment:
Use absolute paths while importing.

```suggestion
from wildfirepy.net.util.modis import *
```